backend/app/api/ws/__init__.py

```
@@ -11,5 +11,13 @@
 from app.api.ws.events import *  # noqa: F401,F403
+from app.api.ws.local_executor_namespace import (
+    get_local_executor_namespace,
+    register_local_executor_namespace,
+)
 
-__all__ = ["register_chat_namespace"]
+__all__ = [
+    "register_chat_namespace",
+    "register_local_executor_namespace",
+    "get_local_executor_namespace",
+]
```

Ruff (0.14.14)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)